src/controller/CHIPDeviceController.cpp

```diff
@@ -1071,9 +1091,10 @@ CHIP_ERROR DeviceCommissioner::ProcessCSR(DeviceProxy * proxy, const ByteSpan &
     MATTER_TRACE_EVENT_SCOPE("ProcessOpCSR", "DeviceCommissioner");
     VerifyOrReturnError(mState == State::Initialized, CHIP_ERROR_INCORRECT_STATE);
 
-    ChipLogProgress(Controller, "Getting certificate chain for the device from the issuer");
+    // This will be used once the GenerateNOCChain
```

Comment:
This comment seems to be missing its end.

Reply:
Stale bit from the first commit - removed now.